Assets/scss/components/quesionnaire.scss

```diff
@@ -0,0 +1,276 @@
+.cmp-questionnaire {
+  max-width: toRem(1175);
+  margin: auto;
+
+  @include max-screen(sm) {
+    max-width: toRem(767);
+    margin: auto;
+  }
+
+  @include breakpoint-between(sm, md) {
+    max-width: toRem(1199);
+    margin: auto;
+  }
+
+  &__container {
+    display: flex;
+    background-color: $grey94;
+    height: toRem(590);
+
+    @include max-screen(sm) {
+      display: flex;
+      flex-direction: column;
+      height: 164vh;
+      overflow: auto;
+      background-color: $grey94;
+    }
+
+    @include breakpoint-between(sm, md) {
+      height: 100vh;
+      overflow: auto;
+      display: flex;
+      flex-direction: column;
+    }
+  }
+
+  &__image {
+    max-width: 50%;
+    height: toRem(590);
+
+    @include max-screen(sm) {
+      max-width: 100%;
+      object-fit: cover;
+    }
+
+    @include breakpoint-between(sm, md) {
+      max-width: 100%;
+      object-fit: cover;
+    }
+  }
+
+  &__form {
+    margin-block-start: toRem(40);
+    margin-inline: toRem(40) toRem(50);
+
+    @include breakpoint-between(sm, md) {
+      margin-inline-end: toRem(100);
+    }
+  }
+
+  &__title {
+    text-align: left;
```

Comment:
`text-align` also can use newer "logical" directions like [start](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#start) in case there's RTL language it will be on the right side.

Reply:
Thanks for the Feedback this has been addressed have changed it